e2e/benchmarks/browserstack-benchmark/app.js

```diff
@@ -362,14 +362,21 @@ function setupHelpMessage() {
     help: 'runs GCP compatible version of benchmarking system',
     action: 'store_true'
   });
-  parser.add_argument('--weeklyCycleRun', {
+  parser.add_argument('--period', {
     help: 'runs a part of models specified in --benchmarks\'s file in a ' +
-      'weekly cycle and the part of models to run is determined by the day ' +
-      'of a week. The value could be -1 (the day will the day at the ' +
-      'runtime) or 0~6 (representing Sunday to Saturday).',
+      'cycle and the part of models to run is determined by the date ' +
+      'of a month. The value could be 1~31.',
     type: 'int',
     action: 'store'
   });
+  parser.add_argument('--date', {
+    help: 'set the date for selecting models and this works only if period ' +
+      'is set. The value could be -1 (the day will the date at the runtime) ' +
+      'or 1~31.',
+    type: 'int',
+    default: -1,
```

Comment:
Instead of default, you can make this argument optional and it will be undefined if it's not passed.

Reply:
Done.